python/cudf/cudf/tests/text/test_text_methods.py

```diff
@@ -878,27 +878,38 @@ def test_jaccard_index():
         str1.str.jaccard_index(str3, 5)
 
 
-def _make_list_of_strings_of_random_length(num_strings, max_length):
+def _make_list_of_strings_of_random_length(
+    num_strings, min_length, max_length
```

Comment:
Maybe require kwargs?
```suggestion
    num_strings, *, min_length, max_length
```